lib/tbd/psi.rb

```diff
@@ -1762,13 +1762,8 @@ def process(model = nil, argh = {})
             farthest_V = origin_point_V if farther
           end
 
-          angle = reference_V.angle(farthest_V)
-
-          if angle.nil?
-            invalid("#{id}: duplicate vertices (or too narrow)", mth, 0, ERR, 0)
-            angle = 0
-          end
-
+          angle  = reference_V.angle(farthest_V)
+          angle  = 0 if angle.nil?
```

Comment:
There are at least 2 circumstances that can trigger a nilled polar angle (around an edge):

 1. a "tiny" (i.e. invalid) surface, e.g. the width of one's finger
 2. when edges are "leader lines", as briefly discussed [here](https://github.com/rd2/osut/pull/17#discussion_r1530915726)

Getting the polar angle right allows TBD to distinguish concave vs convex corners, for instance. But in other cases, it has no effect (e.g. _transition_ edges). The raised error was initially justified when dealing with bad geometry (something the user should remedy), yet "leader lines" (i.e. _transition_ edges) shouldn't raise such an error. Simpler here to just mute the error entirely.